ez_setup.py

```diff
@@ -1,426 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Hmm does this mean we can stop shipping this file with motor, pymongo etc as well? I wondered about this myself when tagging Motor 2.2 but all I could learn by reading docs on pypa was that this file bootstraps setuptools into environment where it isn't installed to begin with. AFAICT this can still be the case with Python 3 environments so I let it be.

Reply:
I don't think you want to delete this file until we no longer support 2.7. ensurepip came along in Python 2.7.9.

Reply:
So we can stop shipping this in Python 3 even though it is still possible, albeit unlikely/an antipattern, to create virtualenvs without `setuptools` in Python 3?

Reply:
Ah, I see https://docs.python.org/3/library/ensurepip.html

Reply:
Added it back.